addons/web/static/src/views/list/list_renderer.js

```diff
@@ -287,13 +287,16 @@ export class ListRenderer extends Component {
         return Object.values(list.fields)
             .filter(
                 (field) =>
+                    list.activeFields[field.name] &&
```

Comment:
Why we need to add this check ? Strange to have unknown field ?

Reply:
The properties can exist in the `fields` object but not yet loaded by the model.